runtime/codert_vm/jswalk.c

```diff
@@ -838,13 +841,15 @@ jitWalkResolveMethodFrame_walkF(J9StackWalkState *walkState, UDATA ** pendingSen
 		UDATA fpParmNumber = J9SW_JIT_FLOAT_ARGUMENT_REGISTER_COUNT - *floatRegistersRemaining;
 #ifdef J9VM_INTERP_STACKWALK_TRACING
 		if (walkState->flags & J9_STACKWALK_ITERATE_O_SLOTS) {
+			assert(NULL != walkState->jitFPRegisterStorageBase);
```

Comment:
Same, this can be moved above the if check, then you won't need the second assert in this function.
also `Assert_CodertVM_true`